sdk/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpansProcessor.java

```diff
@@ -151,126 +147,6 @@ public void forceFlush() {
     worker.forceFlush();
   }
 
-  /**
-   * Returns a new Builder for {@link BatchSpansProcessor}.
-   *
-   * @param spanExporter the {@code SpanExporter} to where the Spans are pushed.
-   * @return a new {@link BatchSpansProcessor}.
-   * @throws NullPointerException if the {@code spanExporter} is {@code null}.
-   */
-  public static Builder newBuilder(SpanExporter spanExporter) {
-    return new Builder(spanExporter);
-  }
-
-  /** Builder class for {@link BatchSpansProcessor}. */
-  public static final class Builder {
-
-    private static final long SCHEDULE_DELAY_MILLIS = 5000;
-    private static final int MAX_QUEUE_SIZE = 2048;
```

Comment:
What happened to these default values? Doing a quick pass, I don't see them anywhere and don't know whether they exist anymore in this PR.

Reply:
They have been moved to the [`Config` class](https://github.com/dynatrace-oss-contrib/opentelemetry-java/blob/config/sdk/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpansProcessor.java#L315)